core/src/calleeFunctions/CurveLpTokenUniv3Callee.ts

```diff
@@ -12,10 +12,10 @@ const getCalleeData = async function (
     collateral: CollateralConfig,
     profitAddress: string
 ): Promise<string> {
-    if (collateral.exchange.callee !== 'CurveLpTokenUniv3Callee') {
+    if (!collateral.exchanges.hasOwnProperty('Curve Token V3')) {
```

Comment:
If you properly extend the type, you don't need to change this line

Reply:
Yes, you're right, but I meant the usage of `hasOwnProperty` which depends on the arbitrary key name instead of the exact callee name